packages/ses/src/module-instance.js

```diff
@@ -4,6 +4,77 @@ import { create, entries, keys, freeze, defineProperty } from './commons.js';
 // q, for enquoting strings in error messages.
 const q = JSON.stringify;
 
+export function makeThirdPartyModuleInstance(
+  compartmentPrivateFields,
+  staticModuleRecord,
+  compartment,
+  moduleAliases,
+  moduleSpecifier,
+  resolvedImports,
+) {
+  const { exportsProxy, proxiedExports, activate } = getDeferredExports(
+    compartment,
+    compartmentPrivateFields.get(compartment),
+    moduleAliases,
+    moduleSpecifier,
+  );
+
+  const notifiers = create(null);
+
+  if (staticModuleRecord.exports) {
```

Comment:
Why allow the `exports` to be omitted? Is the semantics of omitting it any different from providing one whose value is an empty array? I don't see any. Unless there's a logic reason for keeping it, I'd prefer to prohibit this case.

Reply:
Whatever we choose, it should apply equally well to `imports` and `reexports` (export * from _). I fell okay about requiring an empty `imports` array, and less okay about requiring a `reexports` property. But…still okay.